ChatApp.xcodeproj/project.pbxproj

```diff
@@ -127,6 +155,54 @@
 			path = ChatAppUITests;
 			sourceTree = "<group>";
 		};
+		8FF84BFF29F94C3000A815E8 /* View */ = {
```

Comment:
View Controller is not a view, it will be better to name folder as a "Controllers"